addons/binding/org.openhab.binding.zway/README.md

```diff
@@ -190,4 +190,4 @@ Because textual configuration isn't useful, follow the instructions in the [Gett
 -   Only polling is available. Further versions will contain other mechanisms under usage of the WebSocket implementation of Z-Way or MQTT.
 
 <br>
-<img src="doc/BMWi_4C_Gef_en.jpg" width="200">
+<img src="https://github.com/openhab/openhab2-addons/raw/master/addons/binding/org.openhab.binding.zway/doc/BMWi_4C_Gef_en.jpg" width="200">
```

Comment:
Although I'm not sure about this. I think github wants people to use raw.githubusercontent.com

Reply:
Please try replacing it by the Markdown syntax:
```
![BMWi](./doc/BMWi_4C_Gef_en.jpg)
```
I believe relative vanilla HTML references don't work because of webpack handling the assets.
See https://v0.vuepress.vuejs.org/guide/assets.html#relative-urls.

Reply:
We should then check for other links that use "doc" folders, because so far this has been the official way to include resources in the readmes...

Reply:
done